src/components/Cart/Cart.jsx

```diff
@@ -1,54 +1,46 @@
 import React from 'react'
 import "./Cart.css"
 
-function Cart() {
+function Cart({item,cardQtyHandler,removeFromCartHandler}) {
+    console.log("cart card se",item)
+    console.log(item.qty)
+    console.log()
     return (
-        <div className="main_card_container">
+        // <div className="main_card_container">
             <div className="card_container">
                 <div className="card_wrapper">
-                    <img className="card_image" src="./Assets/card.jpg" alt="cart_product" />
+                    <img className="card_image" src={item.image} alt="cart_product" />
                     <span className="card_badge">New</span>
                 </div>
                 <div className="card_info">
-                    <div className="card_title">Men's Premium Wallet</div>
+                    <div className="card_title">{item.title}</div>
                 </div>
                 <div className="card_price">
-                    <span className="selling_price">₹599</span>
-                    <small className="cost_price">₹1199</small>
-                    <small className="discount">50% off</small>
+                    <span className="selling_price">₹{item.price}</span>
+                    <small className="cost_price">₹{item.original_price}</small>
+                    <small className="discount">{item.discount}% off</small>
                 </div>
                 <div className="quantity">
                     <p className="cart_quantity">Quantity: </p>
-                    <button className="quantity_decrease">-</button>
-                    <input className="quantity_box" type="text" />
-                    <button className="quantity_increase">+</button>
+                    {item.qty === 1 ? 
+                    <button disabled="true" className="quantity_decrease" 
+                        onClick={()=>cardQtyHandler(item._id,"decrement")}>-</button> : 
```

Comment:
You're using this function multiple times, so better if you extract it out.

Reply:
I didnt understood.. Can you please elaborate a little bit ?